### ISSUE TITLE
Add "Contact if issues" to homepage

### ISSUE THREAD

**Issue by seekingsnowflakes** [bradnoble/msc-vuejs#98]
Add a note for who to contact if a bug is found. This could also be in an FAQ document but would be better to have on the main page.

**Comment by idevtech** [comment]
Agreed. What we need to figure out is how to word it. I would not recommend putting an email address on the site or it will quickly end up in a spammer's DB.
